src/test/java/PageObjects/HomePage.java

```diff
@@ -2,9 +2,11 @@
 
 public class HomePage extends BasePage{
 
-
     public void getHomePage() {
-        String url = "http://3.11.77.136/index.php";
-        response = sendGetRequest(url);
```

Comment:
Could peeps remind me I need to ask @matthewmccormick matt about this later? It is to do with possibly making a method to get and page on the website and placing it in basepage or having multiple methods in different PageObjects.